web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -191,6 +201,23 @@ class WebViewState(webContent: WebContent) {
      * For more fine grained control use the OnError callback of the WebView.
      */
     val errorsForCurrentRequest = mutableStateListOf<WebViewError>()
+
+    /**
+     * Sealed class for constraining possible loading states.
+     * See [Started] and [Finished].
+     */
+    sealed class LoadingState
```

Comment:
Let's move this out of the `WebViewState` class and match the implementation of `WebContent` for consistency with the child classes being inside the sealed class. I think a better name as well might be `Loading` instead of `Started`. What do you think?

Reply:
Yea, I was tossing up between Loading and Started. Initially it was Loading, but I changed it to better reflect where it's set from. Happy to change it back though.